tests/pebble.rs

```diff
@@ -88,6 +88,48 @@ async fn tls_alpn_01() -> Result<(), Box<dyn StdError>> {
         .await
 }
 
+/// Test subproblem handling by trying to issue for a forbidden identifier.
```

Comment:
Nit: a trailing dot slipped through (thanks for getting it right on many of the others).

Reply:
:blush:  Fixed!